default.nix

```diff
@@ -1,40 +1,33 @@
-{ repo ? builtins.fetchGit ./.
-, versionFile ? ./.version
-, officialRelease ? false
-
-, nixpkgs ? null
-, config ? {}
-, system ? builtins.currentSystem
+{ stdenv
+, lib
+, perl
+, nix
+, glibcLocales
+, Mojolicious
+, MojoliciousPluginStatus
+, IOSocketSSL
+, DBI
+, DBDSQLite
+, xz
+, bzip2
 }:
 
-with builtins;
-
-let
-  bootstrap = import ./nix/bootstrap.nix {
```

Comment:
Sorry for this radical change. I needed to get something working quick and this was the easiest option.

Reply:
In long term a lot of these git version information can be taken from the flake itself.